images/macos/macos-12-Readme.md

```diff
@@ -58,29 +54,29 @@
 - bazelisk 1.11.0
 - bsdtar 3.5.1 - available by 'tar' alias
 - Curl 7.82.0
-- Git 2.35.1
+- Git 2.35.3
```

Comment:
@igorboskovic3 @zoran-bjelica let's wait for the confirmation here
https://github.slack.com/archives/CNS9X1LHJ/p1650296111460829?thread_ts=1649821793.741599&cid=CNS9X1LHJ

Reply:
Canceled deployment. See https://github.slack.com/archives/CNS9X1LHJ/p1650296453190269?thread_ts=1649821793.741599&cid=CNS9X1LHJ